packages/core/src/graph/controller/item.ts

```diff
@@ -447,6 +442,9 @@ export default class ItemController {
     children.combos.forEach((c) => {
       graph.hideItem(c, stack);
     });
+    setTimeout(() => {
+      this.updateAncestorCombos(combo as ICombo);
+    }, 0);
```

Comment:
You should use this one instead: https://github.com/sirensolutions/G6/blob/2c326f07eb9300c5fd4675c9034270100ccc9cbf/packages/core/src/graph/graph.ts#L1964-L2018

Note how that one also travels up the combo tree hierarchy. And check if that works without the `setTimeout`, too.